Tests/YCoreUITests/Protocols/SystemImageTests.swift

```diff
@@ -33,6 +33,18 @@ final class SystemImageTests: XCTestCase {
         YCoreUI.isLoggingEnabled = true
     }
     
+    func test_loadImageWithRenderingMode() {
+        RenderMode.allCases.forEach {
+            XCTAssertEqual(
+                $0.loadImage()?.pngData(),
+                UIImage(
+                    systemName: $0.rawValue,
+                    withConfiguration: RenderMode.configuration
+                )?.withRenderingMode(RenderMode.renderingMode ?? .alwaysTemplate).pngData()
+            )
```

Comment:
```suggestion
            XCTAssertEqual($0.image.renderingMode, .alwaysOriginal)
```
Let's just check the returned image's rendering mode.